### PR TITLE
8.3 What's New

### PR DESCRIPTION
**Pull request by Pessimistress**
For #4998

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -38,8 +81,6 @@ Many new features are added to `TileLayer`, `MVTLayer` and `TerrainLayer` to imp
 - `TileLayer`'s `tileSize` prop can be used to fine-tune the zoom level at which tiles are loaded.
 - `TileLayer`'s `renderSubLayers` is now always called after the tile layer is loaded, i.e. `props.data` is never a Promise.
 - `TileLayer` can now be used in multi-view applications, as long as each `TileLayer` instance is rendered into one view. See [documentation](/docs/developer-guide/views.md#rendering-layers-in-multiple-views) for an example.
-- `TileLayer`'s tile requests may now be aborted if there are too many queued or ongoing requests. Note: only tiles that aren't visible will be aborted.
```
Comment (kylebarron) [inline docs/whats-new.md:41]: Why deleted?

**Review by kylebarron — APPROVED** [review]

File: docs/whats-new.md (hunk 1 above)
Comment (Pessimistress) [reply docs/whats-new.md:41]: These are 8.3 features, should not have been added to 8.2 section.

File: docs/whats-new.md [hunk 2]
```diff
@@ -2,9 +2,52 @@
...
+
+Layers with `pickable` disabled (the default) skip generating picking attributes. This can greatly improve initial loading time and memory usage if a large layer does not have to be interactive.
+
+If no layer is pickable, deck no longer creates the picking FrameBuffer to save memory.
+
+`Deck` also added some experimental [optimization options](/docs/developer-guide/tips-and-tricks.md#optimization-for-mobile) to allow more fine-grained control of memory usage on memory-restricted devices.
+
+### pydeck
+
+TODO
```
Comment (Pessimistress) [inline docs/whats-new.md:39]: @ajduberstein

File: docs/whats-new.md [hunk 3]
```diff
@@ -2,9 +2,52 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.3
+
+Release Date: TBD
```
Comment (Pessimistress) [inline docs/whats-new.md:7]: TODO screenshots: CARTO / PathStyleExtension

File: docs/whats-new.md [hunk 4]
```diff
@@ -2,9 +2,80 @@
...
+- `getTileData` now receives an additional argument `signal` that implements [AbortSignal](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal). Tile requests may now be aborted if there are too many queued or ongoing requests.
+- The `getTileData` function passed in to the current layer instance will be called rather than the first layer instance. This allows applications to progressively adjust loading behavior without having to reload all tiles.
+- New `onTileUnload` callback
+- URL templates now support `{-y}` for TMS indexing
+- Experimental support for `GlobeView` in `TileLayer` and `MVTLayer`
+- Improved memory usage in tile caching
+
+### General Performance
+
+Layers with `pickable` disabled (the default) skip generating picking attributes. This can greatly improve initial loading time and memory usage if a large layer does not have to be interactive.
```
Comment (ibgreen) [inline docs/whats-new.md:44]: Does the word "greatly" set the right expectations?

File: docs/whats-new.md [hunk 5]
```diff
@@ -2,9 +2,80 @@
...
+
+If no layer is pickable, deck no longer creates the picking FrameBuffer to save memory.
+
+`Deck` also added some experimental [optimization options](/docs/developer-guide/tips-and-tricks.md#optimization-for-mobile) to allow more fine-grained control of memory usage on memory-restricted devices.
+
+### pydeck 0.5
+
+pydeck 0.5 includes new event handlers — `on_hover`, `on_click`, `on_view_state_change`, and `on_resize` — which can be used to create rich interactive applications in Jupyter. This release works in sync with deck.gl 8.3.
```
Comment (ibgreen) [inline docs/whats-new.md:52]: Link to https://pydeck.gl/CHANGELOG.html 

@ajduberstein are you republishing the pydeck docs?

**Review by ibgreen — APPROVED** [review]
Ship it!

**Review by ajduberstein — APPROVED** [review]

File: docs/whats-new.md (hunk 5 above)
Comment (ajduberstein) [reply docs/whats-new.md:52]: At the moment the best changelog we can currently link to is probably [the source of the docs](https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/docs/CHANGELOG.rst#050b1---aug-23-2020). The gif I've provided for this is from the JupyterCon demo, which is [here](https://github.com/ajduberstein/jupytercon2020/blob/master/Isochrones%20with%20GraphHopper%20and%20pydeck.ipynb), and we could link to that as well.

This needs a dedicated documentation page–I'm writing one and will have it out in the next 2-3 days as I publish pydeck 0.5. I'll update https://pydeck.gl after that.

IMG_7734

**Pull request merged**